packages/create-next-app/package.json

```diff
@@ -42,8 +42,7 @@
     "ci-info": "watson/ci-info#f43f6a1cefff47fb361c88cf4b943fdbcaafe540",
     "commander": "2.20.0",
     "conf": "10.2.0",
-    "cpy": "7.3.0",
-    "cross-spawn": "6.0.5",
+    "cross-spawn": "7.0.3",
     "fast-glob": "2.2.7",
```

Comment:
since `cpy` is removed now so ig u can bump `fast-glob` to v3 as v3 bundle size is less than v2

Reply:
Indeed!
I will have a look at the `fast-glob`'s changelog to see if it would break `create-next-app`.

Reply:
Done in [`09d87d7` (#53241)](https://github.com/vercel/next.js/pull/53241/commits/09d87d7430a4cea92a3a7360fcbbe6a7a53edc49), this removes another 100+ KiB, thanks!